src/common/numformatter.cpp

```diff
@@ -171,6 +171,47 @@ bool wxNumberFormatter::GetThousandsSeparatorIfUsed(wxChar *sep)
 #endif // wxUSE_INTL/!wxUSE_INTL
 }
 
+bool wxNumberFormatter::GetThousandsSeparatorAndGroupingIfUsed(wxChar *sep, wxString *gr)
+{
+#if wxUSE_INTL
+    static wxChar s_thousandsSeparator = 0;
+    static wxString s_grouping;
+    static LocaleId s_localeUsedForInit;
+
+    if ( s_localeUsedForInit.NotInitializedOrHasChanged() )
+    {
+        const wxString
+            s = wxLocale::GetInfo(wxLOCALE_THOUSANDS_SEP, wxLOCALE_CAT_NUMBER);
+        if ( s.length() == 1 )
```

Comment:
Note to self: need to check whether this works correctly with non-ASCII separators. as in `fr_FR.utf8` locale, the thousands separator is 'NARROW NO-BREAK SPACE' (U+202F).